metacatalog/models/entry.py

```diff
@@ -524,7 +524,12 @@ def details_table(self, fmt='html'):
         # get the details
         details = dict()
         for key, detail in self.details_dict(full=True).items():
+            # nested details
             if isinstance(detail['value'], dict):
+                # include top-level detail of nested detail
+                details[key] = detail.copy()
+                details[key]['value'] = 'nested'
```

Comment:
I suggest to not use the literal `nested`, but something like `__nested__`, to make clear, that `nested` is **not** the actual value of the detail. Maybe you have an better idea than `__nested__`...

Reply:
I will use `'-'` instead of `'nested'` here.

Reply:
I like